### PR TITLE
hello-world java spark using kn and tabs

### PR DESCRIPTION
**Pull request by csantanapr**
## Proposed Changes

- hello-world java spark using kn and tabs

### CODE REVIEW

File: docs/serving/samples/hello-world/helloworld-java-spark/index.md [hunk 1]
```diff
@@ -5,4 +5,188 @@ weight: 1
...
+   ```shell
+   # Build the container on your local machine
+   docker build -t {username}/helloworld-java .
+   # Push the container to docker registry
+   docker push {username}/helloworld-java
+   ```
+
+## Deploy
+
+1. After the build has completed and the container is pushed to docker hub, you
```
Comment (mpetason) [inline docs/serving/samples/hello-world/helloworld-java-spark/index.md:78]:
```suggestion
1. After the build has completed and the container is pushed to Docker Hub, you
```
Capitalized for consistency

**Comment by csantanapr** [comment]
@mpetason can you check if the CLA problem is with the email you use for the joined commit?

**Comment by csantanapr** [comment]
Anything else that is blocking this PR?

**Comment by abrennan89** [comment]
@csantanapr I think this CLA thing usually can be fixed by doing a force push, or someone @ Google can manually add the label.

As for reviews, this looks fine to me, but maybe some more technical eyes on it for a review? cc @tcnghia @vagababov could you review please?

**Comment by evankanderson** [comment]
It looks like @mpetason has his visible GitHub address (which is used for the co-authoring of PRs when suggestions are accepted) set to `m...@gmail.com` (see [this commit](https://github.com/knative/docs/pull/2834/commits/e23cf30bcce67d1f3db7004737107c2b051bb1c5)). It looks like his commits (like 6ecf46eee50798cb118b8275116853ccab9e5a40) are made with an `@ibm.com` email, which is probably the one that accepted the CLA.

**Comment by mpetason** [comment]
Yeah I think that's it. I usually author with my IBM email which is what is on the CLA.

**Comment by csantanapr** [comment]
I rebase and squash to fix the CLA

**Comment by csantanapr** [comment]
Thank 💯 @vaikas for merging this

**Pull request merged**